run_dir/design/libpooling_queues.html

```diff
@@ -14,7 +14,6 @@
       <h1><span id="page_title">Library pooling queues</span>
         <small class="text-muted">Showing all Library pooling queues</small>
       </h1>
-      <button type="button" id="libpools_copy_table_btn" class="btn btn-outline-dark"><span class="fa fa-copy"></span> Copy table</button>
       <button type="button" id="libpools_copy_table" class="btn d-none" data-clipboard-target="#libpools_table"></button>
```

Comment:
What does this one do? Seems similar to the one you removed, maybe its not needed either

Reply:
Ok, I'll try it, I can't test it, but since you created the pages I trust you 😄